seed_data/challenges/computer-science.json

```diff
@@ -137,8 +158,13 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre Seguridad Informática",
+      "descriptionEs": [
+        "¡Ya casi completamos el curso de Introducción a la Informática!",
+        "Aprenderemos sobre una de más importante invención del siglo 20 - las hojas de cálculo.",
```

Comment:
Although we talked in Slack, I'll keep adding my comment here because I'm about to finish with this PR.

Regarding this line, I think it should say:
"Aprenderemos sobre una de *las invenciones más importantes* del siglo 20 - las hojas de cálculo."